regolith/schemas.py

```diff
@@ -3514,8 +3515,12 @@
             "description": "Your publication recommendation",
             "required": True,
             "type": "string",
-            "eallowed": ["reject", "asis", "smalledits", "diffjournal",
-                         "majoredits"],
+            "eallowed": ["invited"],
```

Comment:
This should not be changed in the schema.  Why do you think it needs to be changed?

Reply:
Completed. It has been reverted back to its original allowed values. I think it was because a bad merge that happened before, which I didn't notice while cleaning the branch.